src/librustdoc/doctest/runner.rs

```diff
@@ -136,19 +133,23 @@ mod __doctest_mod {{
 
 #[rustc_main]
 fn main() -> std::process::ExitCode {{
-const TESTS: [test::TestDescAndFn; {nb_tests}] = [{ids}];
+let tests = {{
+    let mut tests = Vec::new();
```

Comment:
Can you at least use `with_capacity` since we already have `nb_tests`?

Reply:
Yeah!